test/controllers/address.controller.test.js

```diff
@@ -156,17 +155,17 @@ describe('Address controller', () => {
       describe('when the request succeeds', () => {
         beforeEach(() => {
           Sinon.stub(SubmitSelectAddressService, 'go').returns({
-            redirect: '/system/notices/setup/fecd5f15-bacf-4b3d-bdcd-ef279a97b061/check',
-            succeeded: true
+            redirect: '/system/notices/setup/fecd5f15-bacf-4b3d-bdcd-ef279a97b061/add-recipient'
           })
-          Sinon.stub(AddAdditionalRecipientService, 'go')
         })
 
-        it('redirects to the check page', async () => {
+        it('redirects to the add additional recipient route', async () => {
```

Comment:
I think we can be generic in our test description here.

```suggestion
        it('redirects to the configured route', async () => {
```